antenna/gunicornhooks.py

```diff
@@ -2,6 +2,12 @@
 # License, v. 2.0. If a copy of the MPL was not distributed with this
 # file, You can obtain one at http://mozilla.org/MPL/2.0/.
 
+# Monkey patch stdlib with gevent-friendly bits--need to do this here
+# before we import anything else. Gunicorn does this too late.
+from gevent import monkey
+
+monkey.patch_all()
```

Comment:
The Pub/Sub library uses threads to batch and send asynchronously. We need to gevent patch that before it gets loaded so it's coroutines and not threads. The gunicorn bootstrapping cycle does that too late, hence adding this here.

Reply:
This need is moot now, but I think I'm going to keep this here.